### PR TITLE
Add PostgreSQL feature flag store

### PR DESCRIPTION
**Pull request by manuelxarez**
https://backmarket.atlassian.net/jira/software/c/projects/BRIE/boards/312?modal=detail&selectedIssue=BRIE-4300
As a first pass, let's discuss the implementation within the team before we create a PR upstream
 
- Adds a PostgreSQL store backend for the flipper-client library;
- Also check original discussion https://github.com/carta/flipper-client/issues/44

### CODE REVIEW

File: flipper/contrib/postgresql.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    def __init__(
+        self,
+        conninfo: str,
+        table_name: str = "feature_flags",
+    ) -> None:
+        self._conninfo = conninfo
+        self._table_name = sql.Identifier(table_name)
+        self._name_column = sql.Identifier("name")
+        self._item_column = sql.Identifier("item")
+        self._run_migrations()
```
Comment (gpouilloux) [inline flipper/contrib/postgresql.py:34]: Well, so basically migrations would only be executed when instantiating a `PostgreSQLFeatureFlagStore` which should be done only once for every app start, right?

From the BackRepair codebase:

```
features = FeatureFlagClient(PostgresqlFeatureFlagStore())
```

**Review by gpouilloux — APPROVED** [review]
Looks very promising @manuelxarez

File: flipper/contrib/postgresql.py (hunk 1 above)
Comment (manuelxarez) [reply flipper/contrib/postgresql.py:34]: Yep :)
Of course, nothing stops a client from instantiating the store several times, but even then I hope it's not a big overhead (just one statement for now after all)

**Review by peter-suchowacky — APPROVED** [review]

**Review by kuddel-muddel — APPROVED** [review]
Cool stuff

**Pull request merged**